app/src/main/kotlin/no/nav/tiltakspenger/vedtak/service/behandling/BehandlingServiceImpl.kt

```diff
@@ -123,32 +122,23 @@ class BehandlingServiceImpl(
         }
     }
 
-    override fun startBehandling(behandlingId: BehandlingId, saksbehandler: String) {
-        val behandling = hentBehandlingEllerKastException(behandlingId)
-
-        if (behandling.erÅpen()) {
-            check(behandling.saksbehandler == null) { "Denne behandlingen er allerede tatt" }
-        }
-
-        if (behandling is BehandlingTilBeslutter) {
-            // TODO: Har jeg gjort en glipp, eller var denne checken alltid overflødig?
-            // check(behandling.saksbehandler != null) { "Kan ikke starte å beslutte en behandling uten saksbehandler" }
-            check(behandling.beslutter == null) { "Denne behandlingen har allerede en beslutter" }
-        }
-
-        behandlingRepo.lagre(behandling.startBehandling(saksbehandler))
+    // TODO: Burde det vært to ulike funksjoner avhengig av om det er saksbehandler eller beslutter det gjelder?
+    override fun taBehandling(behandlingId: BehandlingId, utøvendeSaksbehandler: Saksbehandler) {
+        val behandling = hentBehandling(behandlingId)
+        behandlingRepo.lagre(behandling.startBehandling(utøvendeSaksbehandler))
```

Comment:
Jeg tenker; ja!
Jeg kan se for meg at det blir lettere å forstå koden om man har én metode for saksbehandler og én for beslutter 👍